torchtext/experimental/datasets/raw/sequence_tagging.py

```diff
@@ -58,47 +59,43 @@ def _setup_datasets(dataset_name, separator, root=".data"):
         "valid": _construct_filepath(extracted_files, "dev.txt"),
         "test": _construct_filepath(extracted_files, "test.txt")
     }
-
-    datasets = []
-    for key in data_filenames.keys():
-        if data_filenames[key] is not None:
-            datasets.append(
-                RawTextIterableDataset(dataset_name, NUM_LINES[dataset_name],
-                                       _create_data_from_iob(data_filenames[key], separator)))
-        else:
-            datasets.append(None)
-
-    return datasets
+    return tuple(RawTextIterableDataset(dataset_name, NUM_LINES[dataset_name], _create_data_from_iob(data_filenames[item], separator)) if data_filenames[item] is not None else None for item in data_select)
```

Comment:
nit: a newline could make this more readible

Reply:
Fixed.